server/apps/templates/content_templates.py

```diff
@@ -119,6 +126,60 @@ def get_scheduled_templates(self, now):
         return self.find(query)
 
 
+class ContentTemplatesApplyResource(Resource):
+    endpoint_name = 'content_templates_apply'
+    resource_title = endpoint_name
+    schema = item_schema()
```

Comment:
imo it would be easier to use if url is just `content_templates_apply` and there is template and item in schema..